### ISSUE TITLE
Sorting Surnames in Browse by Author in CBSS: Currently the entries are not being sorted correctly by language

### ISSUE THREAD

**Issue by davidamichelson** [srophe/syriaca#203]
Related to issue #184 

Currently the entries are not being sorted correctly by language.

@wsalesky Can you remind me, do you use some sort of unicode test on the characters to sort them or how does it currently test what script is being used?

Here are the ranges for the new tabs:
Greek and Coptic: U+0370–U+03FF (117 characters)
Cyrillic: U+0400–U+04FF, 256 characters
Armenian: U+0530-U+058F (96 code points)
Hebrew: U+0590 to U+05FF and from U+FB1D to U+FB4F

Also, I suppose we should create letter tabs for Greek and Cyrillic where we have alot of entries. For the others we can leave them as is without. Do you need us to provide those alphabets?

Lastly, shall we delete the other tab?

Thanks,
Dave

**Comment by wsalesky** [comment]
@davidamichelson YEs we are using a unicode test. If we remove the other tab, than anything that does not explicitly match the above codes will not show up at all. So it could be problematic. I will run a test version.
